evergreen.yml

```diff
@@ -398,11 +406,16 @@ task_groups:
 buildvariants:
   - name: windows-64
     display_name: "Windows"
-    run_on: [windows-64-vs2019-large]
     tasks:
       - name: build
+        run_on: [windows-64-vs2019-large]
       - name: test-integration-group
+        run_on: [windows-64-vs2019-large]
       - name: sign
+        run_on: [windows-64-vs2015-large]
```

Comment:
How onerous to have to add all the other things for this one change.

Reply:
Technically, we could have a different build variant for signing and leave this one alone for the other tasks.